docs/aspects.md

```diff
@@ -0,0 +1,78 @@
+# All About Aspects
+
+Aspects codify features of the world, the environment, the situation, and your character. They are simple, plain truths that describe both the physical and the abstract. 
+
+There are two primary ways to use Aspects: Invocation and Compels.
+
+## Clarifications
+
+### "Aspects Are True"
+
+Because Aspects Are True, any object or condition described by an Aspect carries narrative weight and sometimes mechanical impact. If you have taken a Consequence for a ***Broken Hand***, it's not just for flavor; you probably cannot handwrite a letter. 
+
+Likewise, any significant setpieces or rules may become codified as an Aspect with the GM's discretion. Though, not every single little thing needs to become an Aspect. 
+
+Of course, this also means that once an Aspect *stops* being true, the Aspect (and any of its Free Invokes) can disappear. If a building was ***On Fire*** but no longer is, then you can no longer invoke that Aspect.
+
+### Boosts 
+
+Boosts are temporary type of Aspect usually granted as a bonus or a consolation under the [Four Actions](Four-Actions)' outcomes. Boosts have 1 Free Invoke, but disappear as soon as you use them.
+
+Thus, Boosts are best named for temporary things, such as a ***Burst of Energy*** or ***Dropped Guard***.
+
+## Using Aspects
+
+### Invoking Aspects
+
+When an Aspect would reasonably factor into an Action, you may Invoke it for 1FP. 
+
+An Aspect may have "Free Invokes", temporary points attached to an Aspect so that you do not need to spend Fate Points. They typically result from Creating An Advantage.
+
+One major benefit of Free Invokes is that they can be stacked, even across Aspects and Boosts, so long as it is narratively justifiable. So, if you have 2 Free Invokes on one Aspect and 1 Free Invoke on another, and you can spend as many or as few of those 3 as you like on one Action.
+
+However, you can only spend 1FP toward Invocation per Action, though you can spend that in addition to Free Invokes.
+
+??? warning
+    Invoking an enemy's Character Aspects can be a valuable strategy, but it comes with a significant risk: "hostile invokes" paid with Fate Points can grant that Fate Point to the character whose Aspect is being invoked!
+
+#### What Invocation Can Do:
+
+1. Add +2 per invoke to your own Roll.
+2. Add +2 per invoke to someone else's Roll against Passive Opposition
+3. Add +2 per invoke to someone or something's Passive Opposition.
+4. Reroll all four of your Fate Dice. (Recommended most for negative rolls)
+
+Fate Points can famously be spent to "Declare A Story Detail", but this is not an Invocation.
+
+### Compelling Aspects
+
+!!! tip "You're doing WHAT now?"
+    **Compelling is not Invoking!!!!!!!!!!**
+
+Compels are used to complicate the players' fates by drawing upon the natural impliations of Aspects: "Because x, shouldn't y?"
```

Comment:
NPCs are compellable!
```suggestion
Compels are used to complicate a character's fates by drawing upon the natural impliations of Aspects: "Because x, shouldn't y?"
```